package.json

```diff
@@ -30,6 +30,7 @@
     "dapp-chain": "GANACHE_ARGS='-b 2' concurrently -k -n ganache,dapp -p '[{time}][{name}]' 'yarn ganache:start' 'sleep 5 && yarn dapp'",
     "forwarder": "node ./development/static-server.js ./node_modules/@metamask/forwarder/dist/ --port 9010",
     "dapp-forwarder": "concurrently -k -n forwarder,dapp -p '[{time}][{name}]' 'yarn forwarder' 'yarn dapp'",
+    "jest:node": "node --inspect ./node_modules/.bin/jest --watch",
```

Comment:
@legobeat 

that's a good point with the implicit `node` entrypoint. The difference with this script is that it spins up the node inspector. 

I originally used `jest:node` because `jest` alone also runs the test. 

I can see how this can be more intuitive though. Thanks for the suggestion! I would say it's in the same scope. Updated https://github.com/MetaMask/metamask-extension/pull/20683/commits/8b6f1954249bcf59a5ed1bc6e9e5aa0ee0adcb97